app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/input_stages/TextureStage.kt

```diff
@@ -45,6 +46,6 @@ internal class TextureStage(
 
         // Input framebuffer resolution
         val framebufferResolutionHandle = glGetUniformLocation(program, "resolution")
-        glUniform2f(framebufferResolutionHandle, texture.width.toFloat(), texture.height.toFloat())
+        glUniform2f(framebufferResolutionHandle, getResolution().width.toFloat(), getResolution().height.toFloat())
```

Comment:
Why do we use the screen resolution when using the TextureStage?

Reply:
i completely redid everything, we now use texture width again